package.json

```diff
@@ -55,6 +55,7 @@
     "chai": "^4.2.0",
     "eslint": "^4.19.1",
     "eslint-find-rules": "^3.3.1",
+    "eslint-plugin-jsdom-internal": "link:./scripts/eslint-plugin",
```

Comment:
Eslint doesn't allow to reference in the eslint config local customs rules. Custom rules can be injected by either passing the [`--ruledir` flag](https://eslint.org/docs/user-guide/command-line-interface#--rulesdir), or by defining an eslint plugin package. The main advantage of using a local package over passing a flag to the CLI is that, eslint integration in the code editor will work out of the box without any configuration. This approach is borrowed from the react repo: [`package.json`](https://github.com/facebook/react/blob/master/package.json#L55)

Using [yarn `link:` dependency](https://github.com/yarnpkg/rfcs/blob/8015fd14f572bc444c35f9625eaef0d7d3bb3bf9/implemented/0000-link-dependency-type.md), to create a symbolic link instead of a copy of the local files in the `node_modules` directory.